release-notes-6.0.0.md

```diff
@@ -7,5 +7,9 @@ ${version-number}
 - **[SCMOD-5524](https://portal.digitalsafe.net/browse/SCMOD-5524)** taskData no longer sent as Base64.  
     taskData is no longer Base64 encoded in order to reduce message size. 
 
+#### Breaking Changes
+- **378627**: Workers built using framework version prior to 6.0.0 are no longer supported.
```

Comment:
You mean prior to 5.1.0: https://github.com/WorkerFramework/worker-framework/releases/tag/v5.1.0

Reply:
Ah, true sorry. Updated